app/controllers/data_api/tad_data_exports_controller.rb

```diff
@@ -2,10 +2,58 @@ module DataAPI
   class TADDataExportsController < ActionController::API
     include ServiceAPIUserAuthentication
 
+    def index
+      exports = DataAPI::TADExport.all
+
+      formatted_exports = exports.map do |e|
```

Comment:
nit: can we use `export` instead of `e`?

Reply:
Good call, thank you